cupyx/fallback_mode/fallback.py

```diff
@@ -88,6 +88,33 @@ def __repr__(self):
     def __doc__(self):
         return self._numpy_object.__doc__
 
+    @staticmethod
+    def _is_cupy_compatible(args, kwargs):
+        """
+        Returns False if ndarray is not compatible with CuPy.
+        """
+        for arg in args:
+            if isinstance(arg, ndarray) and arg._class is not cp.ndarray:
```

Comment:
Sometimes arg might be a list of ndarrays.

Reply:
Ah yes, I'll try to do this using `_get_xp_args`.